integration-examples/splunk-otel-databricks/Splunk OpenTelemetry Collector Install.md

```diff
@@ -0,0 +1,121 @@
+# Splunk OpenTelemetry Collector Installation on Databricks
+
+## OpenTelemetry Collector Installation
+
+There are multiple ways to install the splunk opentelemetry collector on databricks and the host infrastructure. An example start-up script has been provided where Splunk Observability Cloud credentials are saved in the Databricks secret store.
```

Comment:
mind adding a link for `An example start-up script` to the script?

Reply:
no problem, great idea!